src/script/auth/page/Login.tsx

```diff
@@ -182,11 +203,28 @@ const Login = ({
     }
   };
 
+  const resendTwoFactorCode = async () => {
+    setIsResendingTwoFactorCode(true);
```

Comment:
you should be able to infer this based off of the redux status
`startSendTwoFactorCode` && `successfulSendTwoFactorCode`

Reply:
from TM     `case UserActionCreator.successfulSendActivationCode.type: {
      return {...state, activatedEmail: action.payload, sendingCode: false};
    }`

Reply:
https://github.com/wireapp/wire-webapp/pull/12681/commits/feedbbcbeefcb509a19ce4e2b01e0ad51252cc26